tensorboard/webapp/tbdev_upload/tbdev_upload_test.ts

```diff
@@ -76,4 +80,16 @@ describe('tbdev upload test', () => {
       'tensorboard dev upload --logdir {logdir}'
     );
   });
+
+  it('can be closed with button', async () => {
+    const fixture = TestBed.createComponent(TbdevUploadDialogComponent);
+    fixture.detectChanges();
+
+    const copyElement = fixture.debugElement.query(By.css('.close-button'));
+    copyElement.nativeElement.click();
+    fixture.detectChanges();
+    await fixture.whenStable();
```

Comment:
I'm not 100% sure, but is this `await whenStable` required?  If it is, I'm curious to learn more about when it is needed

Reply:
Done. Indeed we did not need these two calls to fixture.